src/Apps/Ingress/Models/Context/NavigationContext.types.ts

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { PageNames } from '../Constants';
```

Comment:
I almost wonder if there should be a types file for the app? or maybe this enum belongs in this file itself? I think that's what I did in OAT but I don't love that the exports for arg types are mingled with the "semi-private" definition of the state. That's just me being picky. I kinda think the args to the actions should be in this file too I think.